contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -769,6 +772,75 @@ contract PayloadIGP95 is PayloadIGPMain {
         IDSAV2(TREASURY).cast(targets, encodedSpells, address(this));
     }
 
+    // @notice Action 13: Update limits for wstETH-ETH, weETH-ETH DEXes and vaults
+    function action13() internal isActionSkippable(13) {
+        {
+            // WSTETH-ETH DEX
+            address WSTETH_ETH_DEX = getDexAddress(1);
+            {
+                DexConfig memory DEX_WSTETH_ETH = DexConfig({
+                    dex: WSTETH_ETH_DEX,
+                    tokenA: wstETH_ADDRESS,
+                    tokenB: ETH_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: true,
+                    baseWithdrawalLimitInUSD: 20_000_000, // $20M
+                    baseBorrowLimitInUSD: 26_000_000, // $26M
+                    maxBorrowLimitInUSD: 51_000_000 // $51M
+                });
+                setDexLimits(DEX_WSTETH_ETH); // Smart Collateral
+            }
+            {
+                // Set max supply and borrow shares
+                IFluidDex(WSTETH_ETH_DEX).updateMaxSupplyShares(9000 * 1e18);
+                IFluidDex(WSTETH_ETH_DEX).updateMaxBorrowShares(8100 * 1e18);
+            }
+        }
+
+        {
+            // WSTETH-ETH T4 vault
+            address WSTETH_ETH__WSTETH_ETH_VAULT = getVaultAddress(44);
+            {
+                VaultConfig memory VAULT_WSTETH_ETH = VaultConfig({
+                    vault: WSTETH_ETH__WSTETH_ETH_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_4,
+                    supplyToken: address(0),
+                    borrowToken: ETH_ADDRESS,
+                    baseWithdrawalLimitInUSD: 15_000_000, // $15M
+                    baseBorrowLimitInUSD: 15_000_000, // $15M
+                    maxBorrowLimitInUSD: 45_000_000 // $45M
+                });
+
+                setVaultLimits(VAULT_WSTETH_ETH); // TYPE_4 => 44
+            }
+        }
+
+        {
+            // weETH-ETH T2 vault
+            address weETH_ETH__wstETH_VAULT = getVaultAddress(74);
+            {
+                // Update max borrow to 45.5M$
+                VaultConfig memory VAULT_weETH_ETH = VaultConfig({
+                    vault: weETH_ETH__wstETH_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_2,
+                    supplyToken: address(0),
+                    borrowToken: ETH_ADDRESS,
+                    baseWithdrawalLimitInUSD: 0,
+                    baseBorrowLimitInUSD: 0,
+                    maxBorrowLimitInUSD: 45_500_000 // 45.5M$
+                });
+
+                setVaultLimits(VAULT_weETH_ETH); // TYPE_2 => 74
+            }
+            {
+                // Update max supply shares
+                IFluidDex(weETH_ETH__wstETH_VAULT).updateMaxSupplyShares(
```

Comment:
must be on dex

Reply:
Also this is missing 

Increase lower range from 0.06% to 0.07% (need to be instant because of the ETH bug)